addons/event/models/event_registration.py

```diff
@@ -252,6 +243,20 @@ def _update_mail_schedulers(self):
     # MAILING / GATEWAY
     # ------------------------------------------------------------
 
+    def _message_compute_subject(self, **kwargs):
+        record_names = dict()
+
+        to_compute_registrations = self.browse([thread_id for thread_id in self.ids if not record_names.get(thread_id)])
```

Comment:
Note that as record_names is initialized hereabove, you don't have to browse as you will anyway get everything. You may use self directly.

Reply:
Addendum: you might receive record_names from parameters actually (I see kwargs) ? In which case effectively you have to filter, but in that case I would use a ``filtered``. You can also use real parameters instead of kwargs then. Not sure if you intend to override record_names from parameters or always force returned results ?

Reply:
Oh, I'm pretty sure I must have copy-pasted the self.browse, the condition at the end should not be there. And of course yes, the browse is unnecessary.

-----------------

The idea here is that I don't actually care what's passed for registrations, I don't use the name of the record and I already know what the name of the model is (being inside an override). So I used a kwargs to show the parameters are not used.

Reply:
Yup... not sure if this is a convention or not, but personally I prefer to keep parameters explicit. But maybe using kwargs is better to highlight we don't use them indeed :) .